### PR TITLE
Localization of errors and fallback components

### PR DESCRIPTION
**Pull request by Fuzznimp**
## Description

This PR adds some localization for the errors messages and fallback components. Web errors will be tackle in the next PR.

Also, I'm necessarily happy with the key naming in there, or with some of the french translation. Any feedback is welcome 😄 .

## Type of change

- [x] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## Checklist

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

**Comment by frieda-ewlines** [comment]
Task linked: [CU-9kywbh [feature] - Localize errors and fallback components](https://app.clickup.com/t/9kywbh)

File: content/locales/fr.ts [hunk 1]
```diff
@@ -148,4 +150,27 @@ export const fr = {
     close: "Fermer",
     logout: "Vous déconnecter",
   },
+  errors: {
+    "404Title": "Nous ne trouvons pas la page que vous cherchez.",
+    "404Content":
+      "Elle peut être expirée, ou il pourrait y avoir une coquille. Peut-être que vous pouvez trouver ce dont vous avez besoin sur notre",
+    homepage: "page d'acceuil.",
```
Comment (Oluane) [inline content/locales/fr.ts:157]:
```suggestion
    homepage: "page d'accueil.",
```

File: content/locales/fr.ts [hunk 2]
```diff
@@ -148,4 +150,27 @@ export const fr = {
...
+    "500Content":
+      "Quelque chose a mal tourné. Nous travaillons à régler ce problème le plus tôt possible.",
+    blankPassword: "Le mot de passe ne peut pas être vide.",
+    passwordMatch:
+      "Votre confirmation de mot de passe ne correspond pas à votre nouveau mot de passe.",
+    passwordCriteria:
+      "Le mot de passe que vous entrez ne répond pas aux critères.",
+    passwordInfo: "Assurez-vous que votre mot de passe contient au moins :",
```
Comment (Oluane) [inline content/locales/fr.ts:165]:
```suggestion
    passwordInfo: "Assurez-vous que votre mot de passe contienne au moins :",
```

File: content/locales/fr.ts [hunk 3]
```diff
@@ -148,4 +150,27 @@ export const fr = {
...
+    "404Content":
+      "Elle peut être expirée, ou il pourrait y avoir une coquille. Peut-être que vous pouvez trouver ce dont vous avez besoin sur notre",
+    homepage: "page d'accueil.",
+    "500Content":
+      "Quelque chose a mal tourné. Nous travaillons à régler ce problème le plus tôt possible.",
+    blankPassword: "Le mot de passe ne peut pas être vide.",
+    passwordMatch:
+      "Votre confirmation de mot de passe ne correspond pas à votre nouveau mot de passe.",
+    passwordCriteria:
+      "Le mot de passe que vous entrez ne répond pas aux critères.",
```
Comment (Meyclem) [inline content/locales/fr.ts:164]:
```suggestion
      "Le mot de passe saisi ne répond pas aux critères.",
```

File: content/locales/en.ts [hunk 4]
```diff
@@ -150,4 +152,25 @@ export const en = {
...
+    "404Title": "We can't find the page you are looking for.",
+    "404Content":
+      "It may have expired, or there could be a typo. Maybe you can find what you need on our",
+    homepage: "homepage",
+    "500Content":
+      "Something went wrong. We are working on getting this fixed as soon as we can.",
+    blankPassword: "Password can't be blank.",
+    passwordMatch:
+      "Your password confirmation does not match your new password.",
+    passwordCriteria: "The password you enter does not meet the criteria.",
```
Comment (Meyclem) [inline content/locales/en.ts:165]:
```suggestion
    passwordCriteria: "The password you entered does not meet the criteria.",
```

**Review by Meyclem — CHANGES_REQUESTED** [review]

**Review by Meyclem — APPROVED** [review]

**Pull request merged**